### ISSUE TITLE
Is Setter/Action removed in 10.0.0?

### ISSUE THREAD

**Issue by paulwongkwan** [JakeWharton/butterknife#1428]
When I use ButterKnife 10.0.0, it show cannot find symbol class Setter. Same happen on Action.

**Comment by contrudar** [comment]
yes, it was removed. You can find it [here](https://github.com/JakeWharton/butterknife/commit/8eb5280c4a7dce8f023d7a29585a2417b82a607e) also you will find that something was moved just to another place like `butterknife.Setter` and `butterknife.Action`. In the javadoc you'll find what to use instead methods like `Butterknife.apply`

**Comment by junlandroid** [comment]
`//        ButterKnife.apply(new View[]{llYears, llDataProductSubmit, tvSoftwareSubmit}, new ButterKnife.Setter<View, Boolean>() {
//            @Override
//            public void set(@NonNull View view, Boolean value, int index) {
//                if (index == 2) {
//                    view.setVisibility(value ? View.GONE : View.VISIBLE);
//                } else {
//                    view.setVisibility(value ? View.VISIBLE : View.GONE);
//                }
//            }
//        }, mDataProductBean != null);

        ViewCollections.set(new View[]{llYears, llDataProductSubmit, tvSoftwareSubmit}, new Setter<View, Boolean>() {

            @Override
            public void set(@NonNull View view, @Nullable Boolean value, int index) {
                if (index == 2) {
                    view.setVisibility(value ? View.GONE : View.VISIBLE);
                } else {
                    view.setVisibility(value ? View.VISIBLE : View.GONE);
                }
            }
        }, mDataProductBean != null);`
